dll/cpl/sysdm/lang/ro-RO.rc

```diff
@@ -34,236 +34,236 @@ END
+    PUSHBUTTON "&Redenumire", IDC_HRDPROFRENAME, 118, 106, 50, 14, WS_DISABLED
+    PUSHBUTTON "Ş&tergere", IDC_HRDPROFDEL, 173, 106, 50, 14, WS_DISABLED
+    GROUPBOX "Secţiunea profiluri hardware", IDC_STATIC, 7, 130, 240, 75
+    LTEXT "Când ReactOS porneşte:", IDC_STATIC, 14, 142, 210, 8
+    AUTORADIOBUTTON "&Aşteaptă până când selectez un profil hardware", IDC_HRDPROFWAIT, 14, 158, 190, 8, WS_GROUP
+    AUTORADIOBUTTON "&Selectează primul profil listat dacă nu selectez un profil în", IDC_HRDPROFSELECT, 14, 173, 190, 8
     LTEXT "secunde", IDC_STATIC, 65, 187, 35, 8
     EDITTEXT IDC_HRDPROFEDIT, 25, 185, 35, 12
     CONTROL "", IDC_HRDPROFUPDWN, "msctls_updown32", UDS_SETBUDDYINT | UDS_ALIGNRIGHT |
             UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS | WS_CHILD | WS_VISIBLE, 25, 185, 7, 10
-    DEFPUSHBUTTON "Con&firmă", IDOK, 143, 213, 50, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 197, 213, 50, 14
+    DEFPUSHBUTTON "OK", IDOK, 143, 213, 50, 14
+    PUSHBUTTON "Revocare", IDCANCEL, 197, 213, 50, 14
 END
 
 IDD_HARDWAREPROFILE DIALOGEX 0, 0, 212, 188
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "Generale"
+CAPTION "General"
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON IDI_HARDPROF, IDC_STATIC, 10, 10, 18, 20, SS_ICON
     LTEXT "", IDC_HWPPROP_PROFILE, 48, 16, 150, 8
     LTEXT "ID andocare:", IDC_STATIC, 10, 42, 43, 8
     LTEXT "", IDC_HWPPROP_DOCKID, 64, 42, 135, 8
-    LTEXT "Număr serial:", IDC_STATIC, 10, 55, 52, 8
+    LTEXT "Număr de serie:", IDC_STATIC, 10, 55, 52, 8
     LTEXT "", IDC_HWPPROP_SERIAL, 64, 55, 135, 8
     GROUPBOX "", IDC_STATIC, 10, 69, 192, 61
-    AUTOCHECKBOX "Acesta este un calculator porta&bil", IDC_HWPPROP_PORTABLE, 17, 68, 120, 10, WS_GROUP | WS_TABSTOP
-    AUTORADIOBUTTON "Sta&rea de andocare este nespecificată", IDC_HWPPROP_UNKNOWN, 22, 85, 160, 10, WS_GROUP | WS_TABSTOP
-    AUTORADIOBUTTON "Calculatorul &este andocat", IDC_HWPPROP_DOCKED, 22, 97, 160, 10
-    AUTORADIOBUTTON "Calculatorul n&u este andocat", IDC_HWPPROP_UNDOCKED, 22, 109, 160, 10
-    GROUPBOX "Selecție profile de dispozitive", IDC_STATIC, 10, 138, 192, 42
-    AUTOCHECKBOX "In&clude întotdeauna acest profil ca opțiune la pornirea ReactOS", IDC_HWPPROP_INCLUDE, 17, 150, 164, 20, WS_GROUP | WS_TABSTOP | BS_MULTILINE
+    AUTOCHECKBOX "Acesta este un computer &portabil", IDC_HWPPROP_PORTABLE, 17, 68, 120, 10, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "&Stare de andocare necunoscută", IDC_HWPPROP_UNKNOWN, 22, 85, 160, 10, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "Computerul este &andocat", IDC_HWPPROP_DOCKED, 22, 97, 160, 10
+    AUTORADIOBUTTON "Computerul este &detaşat", IDC_HWPPROP_UNDOCKED, 22, 109, 160, 10
+    GROUPBOX "Secţiunea profiluri hardware", IDC_STATIC, 10, 138, 192, 42
+    AUTOCHECKBOX "Întotdeauna &include acest profil ca opţiune la pornirea ReactOS", IDC_HWPPROP_INCLUDE, 17, 150, 164, 20, WS_GROUP | WS_TABSTOP | BS_MULTILINE
```

Comment:
pic please of ro-RO MS dlg for the &i accelerator.

Reply:
![2024-05-05 17_56_10-Resource Hacker - sysdm cpl mui](https://github.com/reactos/reactos/assets/113553683/da76ebd7-2f7c-41ef-a66d-c6eae9005661)